buildrunner/__init__.py

```diff
@@ -220,6 +220,9 @@ def __init__(
         # assign back to the global config env for loading files
         self.global_config.env = self.env
 
+        self.single_platform = single_platform or self.global_config.get('single-platform',
```

Comment:
So no way to disable a single platform if it's set in the global file, is that right?

Reply:
You mean, if in the global config it is set to `true` and you want to have the CLI to set it to `false`? Of course, there is a way to do it. But not with the boolean flag 'single-platform' as it is written now, since it is default False when it is not set.